why-polars-for-data-cleaning/index.qmd

```diff
@@ -0,0 +1,185 @@
+---
+title: "Why use Polars for data cleaning?"
+description: "Key to all data storage is good data quality. This is often overlooked as it can be labour intensive, but it is essential for good data analysis. This post explains why we chose to use Polars for data cleaning."
+date: "2024-12-04"
+categories:
+- data cleaning
+- pre-processing data
+---
+
+::: content-hidden
+Use other decision posts as inspiration to writing these. Leave the
+content-hidden sections in the text for future reference.
+:::
+
+## Context and problem statement
+
+::: content-hidden
+State the context and some background on the issue, then write a
+statement in the form of a question for the problem.
+:::
+
+When working with data analysis, data is rarely ready to be used 'as
+is'. There will always be things that are either very wrong (like text
+in what should be a number field), or slightly wrong (like a single
+missing value, or an outlier, which may or may not be a data entry
+error). In Sprout we are not worried about the single missing values or
```

Comment:
```suggestion
in what should be a number field), slightly wrong (like a single
missing or incorrect value), or an outlier potentially due to an error in data entry. 
Since we don't have domain knowledge to know about missing values or
```